job-search-web-scraping.py

```diff
@@ -1,39 +1,88 @@
+import smtplib
+import sys
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from os.path import basename
+from pathlib import Path
+
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
-from pathlib import Path
 
-import sys
+from config import cfg
+
+
+def send_email(sender_email_address, email_password,
+               receiver_email_address, email_subject,
+               email_body, file_path):
+    """
+    sender_email_address: sender email address
+    email_password: sender email password
+    receiver_email_address: receiver email password
+    email_subject: email subject
+    email_body: email body
+    file_path: file to attach in the email
+    """
+    email_smtp = "smtp.gmail.com"
+    # create an email message object
+    message = MIMEMultipart()
+    # configure email headers
+    message['subject'] = email_subject
+    message['From'] = sender_email_address
+    message['To'] = receiver_email_address
+
+    # attach the text file
+    message.attach(MIMEText(email_body))
+    part = MIMEApplication(open(file_path).read())
+    part.add_header('Content-Disposition',
+                    'attachment; filename="%s"' % basename(file_path))
+    message.attach(part)
+
+    # set smtp server and port
+    server = smtplib.SMTP(email_smtp, '587')
+    # identify this client to the SMTP server
+    server.ehlo()
+    # secure the SMTP connection
+    server.starttls()
+    server.ehlo()
+    # login to email account
+    server.login(sender_email_address, email_password)
+    # send email
+    server.send_message(message)
+    # close connection to server
+    server.quit()
 
 
 def indeed_job_search(*args):
-
     browser = None
 
     PATH_TO_GECKO_DRIVER = './geckodriver'
-    PATH_TO_CHROME_DRIVER = './chromedriver'
+    PATH_TO_CHROME_DRIVER = '/usr/local/bin/chromedriver'
 
     if Path(PATH_TO_GECKO_DRIVER).is_file():
         options = webdriver.FirefoxOptions()
         if 'headless' in args:
             options.headless = True
-        browser = webdriver.Firefox(executable_path=PATH_TO_GECKO_DRIVER, options=options)
+        browser = webdriver.Firefox(executable_path=PATH_TO_GECKO_DRIVER,
+                                    options=options)
```

Comment:
Can you reset the formatting of lines that were made shorter from the original source? There's no need to worry about the flake8 error ` E501 line too long`. 
Change this line and all other lines that were made shorter. Thanks.